buildenv/jenkins/jobs/pipelines/Pipeline-OMR-Acceptance

```diff
@@ -33,7 +33,7 @@ def OPENJ9_BRANCH = 'master'
 def OMR_REPO = 'https://github.com/eclipse/openj9-omr.git'
 def OMR_BRANCH = 'master'
 
-TESTS_TARGETS = '_sanity'
```

Comment:
Perhaps a question for @pshipton , when we change this target, will it be `_sanity` (functional and system) or will it be `sanity.functional`? If it is the latter, I think your `default` `case` would need the `if win/9/10` also since the OMR build currently does 8 & 10 on all platforms (including win)

Reply:
I think the answer of this question should not block this PR. By the time we switch OMR build to use AOJ test pipeline, hopefully `win/9/10` issues are fixed. if not, we can add if in default case at that time.